src/main/java/pl/piekoszek/gorskimatches/equation/CharacterChangeInString.java

```diff
@@ -0,0 +1,14 @@
+package pl.piekoszek.gorskimatches.equation;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class CharacterChangeInString {
+    public String changeCharactersInString(String providedSolution, int positionInString, char numberToBeChangedTo) {
```

Comment:
This class can be treated as more generic, and doesn't have to know about existence of any quizzes or solutions, thus method signature could look like that:
`public String change(String string, int position, char character)`

also:
`numberToBeChangedTo` - it can also be '+' or '-' in future
`positionInString` - we know it's in string

Reply:
ok